hw21/test/TestRunner.java

```diff
@@ -0,0 +1,47 @@
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestRunner {
```

Comment:
Sort the test methods in ascending order of priority using Comparator.comparingInt.

Reply:
Processed